.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           - os: macos-latest
             platform: darwin
             arch: arm64
-          - os: windows-latest
+          - os: windows-2022
```

Comment:
windows 2025 server started rolling out yesterday and it seems to cause intermittent failures in the windows virtual audio device script.

something to explore fixing after this pr.